lib/rack/request.rb

```diff
@@ -605,28 +605,48 @@ def split_header(value)
         value ? value.strip.split(/[,\s]+/) : []
       end
 
+      # ipv6 extracted from resolv stdlib, simplified
+      # to remove numbered match group creation.
+      ipv6 = Regexp.union(
```

Comment:
I like this approach. However, should this be a private constants or are we okay with it being a local variable? Either is okay, just wondering out loud.

Reply:
I prefer local variable when we aren't using the value inside a method.